components/radial-chart/src/radial-chart.html

```diff
@@ -0,0 +1,41 @@
+<!-- <div class="dt-radial-chart-wrapper">
+  <svg [attr.viewBox]="_viewBox" [attr.width]="_width">
+    <svg:path
+      *ngIf="_hasBackground"
+      fill="#f8f8f8"
+      [attr.d]="_backgroundPath"
+    ></svg:path>
+    <svg:g
+      dt-radial-chart-path
+      *ngFor="let series of _renderData"
+      [series]="series"
+      [totalValue]="_totalSeriesValue"
+      [overlayTemplate]="_overlay"
+    ></svg:g>
+  </svg>
+  <dt-legend class="dt-radial-chart-legend">
+    <dt-legend-item *ngFor="let series of _renderData">
+      <dt-legend-symbol
+        class="dt-radial-chart-legend-symbol"
+        [style]="series.colorCustomProp"
+      >
+      </dt-legend-symbol>
+      <strong>{{ series.value }}</strong> {{ series.name }}
+    </dt-legend-item>
+  </dt-legend>
+</div> -->
+
+<svg [attr.viewBox]="_viewBox" [attr.width]="_width">
+  <svg:path
+    *ngIf="_hasBackground"
+    fill="#f8f8f8"
```

Comment:
can we move this to a class, or at least give that element a class to be securely selected in the test?